src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
If I was a betting man, my bet would be that this has no significant impact, since the locally scoped `self` reference is orphaned as soon as the operator function returns (provided we don't start passing it into closures). So the lifetime of the object is dictated only by how long external references to the observable persist.

However, as mentioned earlier, I'll do a test in Chrome to backup my handwaving :)